.buildkite/pipeline-resource-definitions/kibana-serverless-release.yaml

```diff
@@ -0,0 +1,38 @@
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: serverless_release
+  description: Initiate kibana serverless releases
+spec:
+  type: buildkite-pipeline
+  owner: 'group:kibana-operations'
+  implementation:
+    apiVersion: buildkite.elastic.dev/v1
+    kind: Pipeline
+    metadata:
+      name: kibana / serverless / release
+      description: Initiate kibana serverless releases
+    spec:
+      env:
+        SLACK_NOTIFICATIONS_CHANNEL: '#kibana-operations-alerts'
+        SLACK_NOTIFICATIONS_ENABLED: 'false'
+      default_branch: main
+      repository: elastic/kibana
+      pipeline_file: .buildkite/pipelines/serverless_deployment/create_deployment_tag.yml
+      provider_settings:
+        build_branches: false
+        build_pull_requests: false
+        publish_commit_status: false
+        trigger_mode: none
+        build_tags: false
+      teams:
+        kibana-release-operators:
+          access_level: MANAGE_BUILD_AND_READ
+      schedules:
+        Weekly automated promotion to QA:
+          cronline: 0 6 * * 1
+          message: Weekly automated promotion to QA
+          env:
+            AUTO_SELECT_COMMIT: 'true'
```

Comment:
We have to verify with the CI team if this is passed along, because I didn't see it mentioned in the RRE schema